app/models/calculated_value.rb

```diff
@@ -11,8 +12,9 @@ def self.populate_for(studies)
         refresh_calculated_values
       end
     rescue StandardError => e
-      puts "Calculated Error: #{e.message}"
-      raise ActiveRecord::Rollback
+      Rails.logger.error("Unexpected error: #{e.message}")
+      Rails.logger.error(e.backtrace.join("\n"))
+      raise
     end
```

Comment:
remove to get more information into airbrake

Reply:
removed